cmd/test-bot.rb

```diff
@@ -104,6 +104,8 @@ class TestBotCmd < AbstractCommand
         comma_array "--skipped-or-failed-formulae=",
                     description: "Use these skipped or failed formulae from formulae steps for a " \
                                  "formulae dependents step."
+        comma_array "--tested-formulae=",
+                    description: "Use these tested formulae from formulae steps for a formulae dependents step."
```

Comment:
If `tested` not equivalent to the detected formulae (or `testing|added|deleted`) without `skipped-or-failed`?

Reply:
Not necessarily, since we filter `testing-formulae` in https://github.com/Homebrew/brew/pull/18166.

See https://github.com/Homebrew/homebrew-core/actions/runs/10597489566/job/29371680254#step:3:77. `baresip` was in the original `--testing-formulae` flag but was filtered out as having no more dependents to test.

After this change (and the companion Homebrew/core PR), `baresip` will not be in `--testing-formulae` but will be in `--tested-formulae`.

Reply:
> Could this be accomplished without a new flag and instead setting the existing flags? If not: why not?

I thought about it, but I couldn't find a simple way to do it.

I guess the key problem is that we need some way to indicate to `test-bot`:
- a list of formulae it needs to test
- a list of formulae it doesn't need to test

We already pass the former information in `--testing-formulae`. The latter sounds a little like `--skipped-or-failed-formulae`, but the way we handle it right now doesn't quite suit.

In particular, making `--skipped-or-failed-formulae` work requires, at least:
1. Taking the set difference between `--testing-formulae` in the formulae job and `--testing-formulae` in the formulae dependents jobs to construct the list of testing formulae to skip, and then passing that to `--skipped-or-failed-formulae`
2. Making `test-bot` deal with two sources of `skipped_or_failed_formulae`, the lists that are written during the formulae build job, and ones passed in the command line

Might be more steps that I haven't worked out yet right now.